nativelib/src/main/scala/scala/scalanative/unsafe/Ptr.scala

```diff
@@ -62,4 +60,12 @@ object Ptr {
 
   @alwaysinline implicit def ptrToCStruct[T <: CStruct](ptr: Ptr[T])(
       implicit tag: Tag[T]): T = !ptr
+
+  @alwaysinline implicit def ptrToCFuncPtr[F <: CFuncPtr](ptr: Ptr[Byte])(
+      implicit tag: Tag.CFuncPtrTag[F]): F =
+    Boxes.boxToCFuncRawPtr(ptr.rawptr).as[F]
```

Comment:
`.as` is going to immediately unwrap the `CFuncRawPtr` that was created by `boxToCFuncRawPtr`. Consider inlining the body of `as` directly here, so that we don't need this transient wrapper.

This also seems to be the only use of `as`, so perhaps it should be removed altogether.

Reply:
Heck, this might be the only remaining use of `CFuncRawPtr` itself! If yes, then we could remove that one altogether, which would be a huge win!